test/utils/assertions.js

```diff
@@ -0,0 +1,11 @@
+var _ = require('lodash');
+var path = require('path');
+var should = require('should');
+should.Assertion.add('equalFileSystemPath', function(value, description){
+    value = value.replace(/\//g, path.sep);
+    if(process.platform == 'win32' && _.startsWith(value, path.sep)){
+        value = __dirname.split(path.sep)[0] + value;
```

Comment:
I'll have to check when I'm home. I thought there was 1 test for which this was needed. 
I think it is safe to assume that the output directory of the tests results is on the same directory as the project + test source.